app/assets/stylesheets/_spacing-units.scss

```diff
@@ -0,0 +1,9 @@
+$su1: 4px;
```

Comment:
Since our color naming uses a `-` to separate the label from the value, would we want to do the same here? For example:

```
$su2 -> $su-2
$su3 -> $su-3
$su4 -> $su-4
```

That would make this more similar to `$smoke-70`, for example.

Reply:
Good catch! I'll update.